---
title: TrioVis
subtitle: Bioinformatics Tool
layout: default
modal-id: 9
date: 2013-05-05
img: triovis.png
thumbnail: triovis-thumbnail.png
alt: image-alt
project-date: May 2013
client: Course work
topic: Bioinformatics, NGS
category: Visual Analytics Tools
link: bioinformatics.oxfordjournals.org/content/early/2013/05/08/bioinformatics.btt267.abstract
description: A visualization apporach for filtering genomic variants of parent-child trios.

---

TrioVis is a visual analytics tool developed for filtering on coverage and variant frequency for genomic variants from exome sequencing of parent-child trios. In TrioVis, the variant data are organised by grouping each variant based on the laws of Mendelian inheritance. Taking three Variant Call Format (VCF) files as input, TrioVis allows the user to test different coverage thresholds (i.e. different levels of stringency), to find the optimal threshold values tailored to their hypotheses, and to gain insights into the global effects of filtering through interaction.

<iframe src="https://player.vimeo.com/video/56494891" width="700" height="450" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>

[Publication](http://bioinformatics.oxfordjournals.org/content/early/2013/05/08/bioinformatics.btt267.abstract). [Git Repository](https://bitbucket.org/vda-lab/triovis/wiki/Home). 